bananza-frontend/styles/globals.css

```diff
@@ -11,6 +21,145 @@ a {
   text-decoration: none;
 }
 
-* {
+*, *:hover, *:focus, :after{
   box-sizing: border-box;
+  transition: cubic-bezier(0.68, -0.55, 0.27, 1.55);
+  transition-duration: 0.4s;
+}
+
+body {
+  background: var(--black);
 }
+
+body,
+html,
+main,
+#__next {
+  width: 100%;
+  height: 100%;
+}
+
+/* Forms */
+form {
+  color: white;
+  display:flex;
+  justify-content:center;
+  align-items:center;
+  flex-direction: column;
+  width:100%;
+}
+
+.formFields{
+  display:flex;
+  justify-content:center;
+  align-items: center;
+  width:100%;
+}
+
+.formFields div{
+  padding:10px;
+}
+
+form div>span {
+  display:table;
+  margin:5px;
+}
+
+label input{
+  display:table;
+}
+
+input {
```

Comment:
Can we do something about the input changing color to plain white on click (focus)? It kinda breaks your nice design, i think it shouldn't do anithing to the background of it or just add a shade or something, not white as it's now.
I think you can find something about this [here](https://stackoverflow.com/questions/66073842/how-do-i-stop-the-input-form-to-change-background-color-and-text-color-when-i-in)

Reply:
Good idea! Improved this a bit, will probably be modified more in the future.